src/us/kbase/test/auth2/lib/storage/mongo/MongoStorageStartUpTest.java

```diff
@@ -151,284 +148,252 @@ public void checkCollectionNames() throws Exception {
 				new Document("v", indexVer)
 						.append("key", new Document("dispcan", 1))
-						.append("name", "dispcan_1")
-						.append("ns", "test_mongostorage.users"),
+						.append("name", "dispcan_1"),
 				new Document("v", indexVer)
 						.append("key", new Document("_id", 1))
-						.append("name", "_id_")
-						.append("ns", "test_mongostorage.users"),
+						.append("name", "_id_"),
 				new Document("v", indexVer)
-						.append("unique", true)	
+						.append("unique", true)
 						.append("key", new Document("idents.id", 1))
 						.append("name", "idents.id_1")
-						.append("ns", "test_mongostorage.users")
 						.append("sparse", true),
 				new Document("v", indexVer)
 						.append("key", new Document("roles", 1))
 						.append("name", "roles_1")
-						.append("ns", "test_mongostorage.users")
 						.append("sparse", true),
 				new Document("v", indexVer)
 						.append("unique", true)
 						.append("key", new Document("user", 1))
-						.append("name", "user_1")
-						.append("ns", "test_mongostorage.users"),
+						.append("name", "user_1"),
 				new Document("v", indexVer)
 						.append("key", new Document("anonid", 1))
 						.append("name", "anonid_1")
-						.append("ns", "test_mongostorage.users")
 						.append("sparse", true)
 				)));
 	}
-	
+
 	@Test
 	public void indexesTestUsers() {
 		final Set<Document> indexes = new HashSet<>();
-		db.getCollection("test_users").listIndexes().forEach((Consumer<Document>) indexes::add);
+		updateIndexes("test_users", indexes);
 		assertThat("incorrect indexes", indexes, is(set(
 				new Document("v", indexVer)
 						.append("key", new Document("custrls", 1))
 						.append("name", "custrls_1")
-						.append("ns", "test_mongostorage.test_users")
 						.append("sparse", true),
 				new Document("v", indexVer)
 						.append("key", new Document("dispcan", 1))
-						.append("name", "dispcan_1")
-						.append("ns", "test_mongostorage.test_users"),
+						.append("name", "dispcan_1"),
 				new Document("v", indexVer)
 						.append("key", new Document("_id", 1))
-						.append("name", "_id_")
-						.append("ns", "test_mongostorage.test_users"),
+						.append("name", "_id_"),
 				new Document("v", indexVer)
 						.append("key", new Document("roles", 1))
 						.append("name", "roles_1")
-						.append("ns", "test_mongostorage.test_users")
 						.append("sparse", true),
 				new Document("v", indexVer)
 						.append("unique", true)
 						.append("key", new Document("user", 1))
-						.append("name", "user_1")
-						.append("ns", "test_mongostorage.test_users"),
+						.append("name", "user_1"),
 				new Document("v", indexVer)
 						.append("key", new Document("expires", 1))
 						.append("name", "expires_1")
-						.append("ns", "test_mongostorage.test_users")
 						.append("expireAfterSeconds", 0L)
 				)));
 	}
+
+	public void updateIndexes(final String name, final Set<Document> indexes) {
+		for (Document index: db.getCollection(name).listIndexes()) {
```

Comment:
```suggestion
	public void getAndUpdateIndexes(final String name) {
	    final Set<Document> indexes = new Set<>();
		for (Document index: db.getCollection(name).listIndexes()) {
```

Now there's no side effect for this method and it simplifies the calling methods

Reply:
:+1: